isubit/iastate_theme
# D8 1564 - Fix infinite growing on WYSIWYG textareas

Right now, as you enter a great deal of content into a WYSIWYG text area, it just keeps growing past the height of the browser window making the editor toolbar stay stuck out of view. For some reason, this bug seems to have been caused (fairly or not) by this theme adding a class to textarea form elements. We were doing that so that we could apply Bootstrap 4's form element styles to textareas.

The workaround is to remove that template, and duplicate the Bootstrap 4 styles manually. 

To test:
1. First, observe the broken behavior. Start adding a lot of text to a WYSIWYG field. 
2. Then use this branch of the theme, clear cache and try again.
3. The WYSIWYG window should grow until it reaches the height of the browser. That way you can still fit the whole field plus the toolbar on the screen.
